front/app/modules/commercial/custom_maps/admin/containers/ProjectCustomMapConfigPage/MapHelperOptions.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+// components
+import MapView from '@arcgis/core/views/MapView';
+import { Box, Button } from '@citizenlab/cl2-component-library';
+import Tippy from '@tippyjs/react';
+
+// utils
+import styled from 'styled-components';
+import messages from './messages';
+
+// hooks
+import { useIntl } from 'utils/cl-intl';
+import useUpdateMapConfig from 'modules/commercial/custom_maps/api/map_config/useUpdateMapConfig';
+
+// types
+import { IMapConfig } from 'modules/commercial/custom_maps/api/map_config/types';
+import { goToMapLocation } from 'components/EsriMap/utils';
+
+const GoToDefaultViewportButtonWrapper = styled.div`
+  position: absolute;
```

Comment:
These are just the existing configuration buttons on the map in the back office.